xray_vision/backend/mpl/cross_section_2d.py

```diff
@@ -329,7 +333,7 @@ def __init__(self, fig, cmap=None, norm=None,
         # Configure the figure in our own image
         #
         #     	  +----------------------+
-        #	      |   H cross section    |
+        #         |   H cross section    |
```

Comment:
Do we need to shift the borders above and below that too?

Reply:
This replaced a tab with spaces, should all be aligned now.